drivers/hmis/lib/form_data/tarrant_county/assessments/caseworker_exit.json

```diff
@@ -0,0 +1,611 @@
+{
+  "item": [
+    {
+      "text": "Client Data",
+      "type": "GROUP",
+      "link_id": "caseworker_exit_client_data",
+      "item": [
+        {
+          "text": "Assessment Date",
+          "type": "DATE",
+          "link_id": "caseworker_assessment_date",
+          "required": true,
+          "mapping": {
+            "field_name": "assessmentDate"
+          },
+          "assessment_date": true
+        },
+        {
+          "text": "Exit Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_date"
+          }
+        },
+        {
+          "text": "Phone Number",
+          "size": "SMALL",
+          "type": "STRING",
+          "link_id": "caseworker_exit_phone",
+          "component": "PHONE",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_phone"
+          }
+        },
+        {
+          "text": "Email",
+          "type": "STRING",
+          "link_id": "caseworker_exit_email",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_email"
+          }
+        },
+        {
+          "text": "Number of weeks in program",
+          "type": "INTEGER",
+          "required": true,
+          "link_id": "caseworker_exit_weeks_in_program",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_weeks_in_program"
+          }
+        },
+        {
+          "text": "Beginning GAS Score",
+          "helper_text": "Target 40.00",
+          "type": "NUMBER",
+          "link_id": "caseworker_exit_beginning_gas_score",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_beginning_gas_score"
+          }
+        },
+        {
+          "text": "Beginning GAS Score Date",
+          "type": "DATE",
+          "link_id": "caseworker_exit_beginning_gas_score_date",
+          "mapping": {
+            "custom_field_key": "caseworker_exit_beginning_gas_score_date"
+          }
+        },
+        {
+          "text": "Exit GAS Score",
+          "helper_text": "Target 40.00",
+          "type": "INTEGER",
```

Comment:
It seems we don't have a FLOAT type yet, only INTEGER, which can cause problems with the form if the users enter non-int values or the calculated values end up being non-integers (see e.g. https://greenriver.slack.com/archives/C0610D9HSSH/p1713471971947969)

This needs to be fixed somehow for this form, but I wasn't sure if the right solution is to make a new FLOAT type now, or just have these as integers and convert the readonly auto-filled fields to strings, or some other route?

Reply:
@martha good question. I ran the export for this one and all the values are integers, they don't actually input fractional values here. So I think we're good to keep this as an integer

Reply:
It can still cause errors if they input only integers, due to the division to come up with the percent change. For example, inputting 39 in the beginning score and 40 in the exit score results in a % change of 2.5, and an error
`Error processing field 'CustomAssessment.caseworker_exit_gas_score_percent_change': unexpected value "2.5"`

I guess we could consider converting the percent CDED type to a string?

Reply:
String sounds like a good idea for storing the percent change. I _think_ we will need a fix [here](https://github.com/greenriver/hmis-warehouse/blob/b89d9d6c73741c007de9adce3da1478899f9f5a4/drivers/hmis/app/models/hmis/hud/processors/base.rb#L251-L258), it should probably check for `Numeric` instead of just `Integer.

Reply:
Decision from dev standup: Ignore persisting the percentage entirely, for now. To do: create ticket to revisit integer/float/percent/currency storage and display